src/forms/ConfirmSignupForm/ConfirmSignupForm.js

```diff
@@ -25,6 +25,7 @@ const ConfirmSignupFormComponent = props => (
         intl,
         onOpenTermsOfService,
         authInfo,
+        idp,
       } = fieldRenderProps;
```

Comment:
Btw. I just noticed that an old typo keeps spreading around. These are not fieldRenderProps but formRenderProps:
https://final-form.org/docs/react-final-form/types/FormRenderProps

It's a bit annoying because it leads to the wrong place when someone tries to debug forms against React Final Form's docs.

(Side note: I think this typo might even be copied Final Form's docs originally.)